### PR TITLE
User POST Route Removed.

### PR DESCRIPTION
**Pull request by JatinAgrawal94**

### CODE REVIEW

**Comment by JatinAgrawal94** [comment]
image
I am getting this error on running: npm run test-integration
I cannot find where is the issue.

**Comment by swarajpure** [comment]
> 
> 
> image
> I am getting this error on running: npm run test-integration
> I cannot find where is the issue.

Please try `sh scripts/tests/testIntegration.sh`
Make sure to read testing guidelines in the CONTRIBUTING.md
We're having a PR which will fix this issue for Windows. It'll be merged soon.

Also, the tests for `POST /users` should also be removed (which are failing, because you've removed the route itself, the tests are unable to proceed with the request)

File: routes/users.js [hunk 1]
```diff
@@ -5,59 +5,7 @@ const usersController = require('../controllers/usersController')
...
- *             schema:
- *               $ref: '#/components/schemas/errors/notFound'
- *       503:
- *         description: serverUnavailable
- *         content:
- *           application/json:
- *             schema:
- *               $ref: '#/components/schemas/errors/serverUnavailable'
- */
-router.post('/', authenticate, userValidator.createUser, usersController.addNewUser)
```
Comment (ankurnarkhede) [inline routes/users.js:57]: @JatinAgrawal94 , can we also remove the tests for this route?

**Review by ankurnarkhede — CHANGES_REQUESTED** [review]
Tests needed to be removed for this route.

File: routes/users.js [hunk 2]
```diff
@@ -5,59 +5,7 @@ const usersController = require('../controllers/usersController')
 const userValidator = require('../middlewares/validators/user')
 
 /**
- * @swagger
```
Comment (ankurnarkhede) [inline routes/users.js:8]: Run generate schema command again for creating schema.

**Review by ankurnarkhede — CHANGES_REQUESTED** [review]

File: routes/users.js (hunk 1 above)
Comment (JatinAgrawal94) [reply routes/users.js:57]: This route allows us creating a new user, making a post request. It needs to be kept.

Comment (swarajpure) [reply routes/users.js:57]: Github OAuth takes care of creating users. Please read the issue carefully: #161 

Any random person can create any dummy account with this route.

**Comment by JatinAgrawal94** [comment]
Hello Repo Owner, Mistakenly In one of my previous PR's I pushed some private data, that needs to be removed , so can you delete that from the history?.

**Comment by ankurnarkhede** [comment]
Issue resolved with another PR.

**Pull request closed without merging**